server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerScanMetrics.java

```diff
@@ -44,8 +49,19 @@ public void addYield(long value) {
     yields.record(value);
   }
 
+  public void incrementOpenFiles(int delta) {
+    Preconditions.checkArgument(delta >= 0, "Parameter must be zero or positive");
```

Comment:
Do we need to throw an unchecked exception here?  Would Math.max(0, val) work instead?

Reply:
I like this better, changed in b1fb539